js/index.js

```diff
@@ -0,0 +1,97 @@
+var url;
+var content = document.getElementById("content");
+var search = document.getElementById("search");
+
+search.addEventListener("click", () => {
```

Comment:
Use an abstraction for the repeated code

Reply:
Didn't think of that before. Will do that now. Thank you

Reply:
Done